src/Operators/vorticity_operators.jl

```diff
@@ -8,35 +8,34 @@
  """
 @inline ζ₃ᶠᶠᶜ(i, j, k, grid, u, v) = Γᶠᶠᶜ(i, j, k, grid, u, v) / Azᶠᶠᶜ(i, j, k, grid)
 
+@inline function ζ₃ᶠᶠᶜ(i, j, k, grid::OrthogonalSphericalShellGrid{FT}, u, v) where FT
+    scaling = ifelse(on_corner(i, j, grid), convert(FT, 4/3), 1)
+    return scaling * Γᶠᶠᶜ(i, j, k, grid, u, v) / Azᶠᶠᶜ(i, j, k, grid)
+end
+
+# Corner
+@inline on_corner(i, j, grid) = on_south_west_corner(i, j, grid) | on_south_east_corner(i, j, grid) | on_north_west_corner(i, j, grid) | on_north_east_corner(i, j, grid)
+
+# South-west, south-east, north-west, north-east corners
+@inline on_south_west_corner(i, j, grid) = ((i == 1) & (j == 1))
+@inline on_south_east_corner(i, j, grid) = ((i == grid.Nx+1) & (j == 1))
+@inline on_north_east_corner(i, j, grid) = ((i == grid.Nx+1) & (j == grid.Ny+1))
+@inline on_north_west_corner(i, j, grid) = ((i == 1) & (j == grid.Ny+1))
+
 #####
 ##### Vertical circulation at the corners of the cubed sphere needs to treated in a special manner.
 ##### See: https://github.com/CliMA/Oceananigans.jl/issues/1584
 #####
 
 """
-     Γᶠᶠᶜ(i, j, k, grid, u, v)
+    Γᶠᶠᶜ(i, j, k, grid, u, v)
 
  The vertical circulation associated with horizontal velocities ``u`` and ``v``.
  """
 @inline function Γᶠᶠᶜ(i, j, k, grid::OrthogonalSphericalShellGrid, u, v)
-    # South-west corner
-    if i == 1 && j == 1
-        return Δy_qᶜᶠᶜ(i, j, k, grid, v) - Δx_qᶠᶜᶜ(i, j, k, grid, u) + Δx_qᶠᶜᶜ(i, j-1, k, grid, u)
-
-    # South-east corner
-    elseif i == grid.Nx+1 && j == 1
-        return - Δy_qᶜᶠᶜ(i-1, j, k, grid, v) - Δx_qᶠᶜᶜ(i, j, k, grid, u) + Δx_qᶠᶜᶜ(i, j-1, k, grid, u)
-
-    # North-west corner
-    elseif i == 1 && j == grid.Ny+1
-        return Δy_qᶜᶠᶜ(i, j, k, grid, v) - Δx_qᶠᶜᶜ(i, j, k, grid, u) + Δx_qᶠᶜᶜ(i, j-1, k, grid, u)
-
-    # North-east corner
-    elseif i == grid.Nx+1 && j == grid.Ny+1
-        return - Δy_qᶜᶠᶜ(i-1, j, k, grid, v) - Δx_qᶠᶜᶜ(i, j, k, grid, u) + Δx_qᶠᶜᶜ(i, j-1, k, grid, u)
-
-    # Not a corner
-    else
-        return δxᶠᵃᵃ(i, j, k, grid, Δy_qᶜᶠᶜ, v) - δyᵃᶠᵃ(i, j, k, grid, Δx_qᶠᶜᶜ, u)
-    end
+    ifelse(on_south_west_corner(i, j, grid) | on_north_west_corner(i, j, grid),
+           Δy_qᶜᶠᶜ(i, j, k, grid, v) - Δx_qᶠᶜᶜ(i, j, k, grid, u) + Δx_qᶠᶜᶜ(i, j-1, k, grid, u),
+           ifelse(on_south_east_corner(i, j, grid) | on_north_east_corner(i, j, grid),
+                  - Δy_qᶜᶠᶜ(i-1, j, k, grid, v) - Δx_qᶠᶜᶜ(i, j, k, grid, u) + Δx_qᶠᶜᶜ(i, j-1, k, grid, u),
+                  δxᶠᵃᵃ(i, j, k, grid, Δy_qᶜᶠᶜ, v) - δyᵃᶠᵃ(i, j, k, grid, Δx_qᶠᶜᶜ, u)))
```

Comment:
Should this last line be:

```suggestion
                  δxᶠᶠᶜ(i, j, k, grid, Δy_qᶜᶠᶜ, v) - δyᶠᶠᶜ(i, j, k, grid, Δx_qᶠᶜᶜ, u)))
```

??

@simone-silvestri, @glwagner?

Reply:
it depends what you are trying to do. `ᶠᶠᶜ` is "boundary aware", `ᶠᵃᵃ` is not.

Reply:
Is there a case that we _don't_ want `ᶠᶠᶜ` over `ᶠᵃᵃ`?

My main question was whether there is a specific reason that in the corner-specific `Γ` method it was `ᶠᵃᵃ` and in the fallback was `ᶠᶠᶜ`.

Reply:
No idea for that.